<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndSettings</class>
 <widget class="QDialog" name="wndSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabMain">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../res/res.qrc">
        <normaloff>:/icons/vkmm.png</normaloff>:/icons/vkmm.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Общие</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTabWidget" name="tabMainInner">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Сеть</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Прокси</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::ExpandingFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Тип:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="proxyType"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Адрес:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLineEdit" name="proxyAddr">
                    <property name="maximumSize">
                     <size>
                      <width>300</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Порт:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="proxyPort">
                    <property name="maximum">
                     <number>65536</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Логин:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="proxyLogin">
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Пароль:</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="proxyPasswd">
                  <property name="maximumSize">
                   <size>
                    <width>300</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Обновления</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCheckBox" name="updatesEnable">
                <property name="text">
                 <string>Проверять обновления программы</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>Уведомления</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Окна подтверждений</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Вы можете сбросить флаги &quot;Больше не спрашивать&quot; для всех окон подтверждения действий</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="resetDontAskAgain">
                  <property name="text">
                   <string>Сбросить</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Кеш</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QCheckBox" name="cacheEnable">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Включить использование кеша на диске</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>При проигрывании аудиозаписи программа будет сохранять её на диске во временное хранилище, обеспечивая мгновенное проигрывание в следующий раз, без загрузки из сети.</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="cacheLimitsGroup">
             <property name="title">
              <string>Настройки кеша</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QCheckBox" name="cacheCountLimit">
                <property name="text">
                 <string>Ограничить хранилище по количеству записей</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Максимум</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="cacheCountLimitSize">
                  <property name="suffix">
                   <string/>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>файлов</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="cacheSizeLimit">
                <property name="text">
                 <string>Ограничить хранилище по общему размеру файлов</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Максимум</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="cacheSizeLimitSize">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Мб</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Внимание: большие значения ограничений могут приводить к замедлению работы программы!</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Состояние кеша</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="cacheCurrentSize">
                <property name="text">
                 <string>?</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="cacheClean">
                  <property name="text">
                   <string>Очистить кеш</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../res/res.qrc">
        <normaloff>:/icons/vk.png</normaloff>:/icons/vk.png</iconset>
      </attribute>
      <attribute name="title">
       <string>VKontakte</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Приватность</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="vkSendStatus">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Отправлять &quot;Я сейчас слушаю&quot; на мою страницу</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../res/res.qrc">:/images/ex-vk-status.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>В ваш статус на странице будет транслироваться текущая аудиозапись (поведение по-умолчанию при прослушивании аудио из самого ВКонтакте).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="vkSendOnline">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Отправлять &quot;Я Онлайн&quot; при прослушивании</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../res/res.qrc">:/images/ex-vk-online.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Выводить вашего пользователя Вконтакте в состояние &lt;b&gt;Online&lt;/b&gt; при запущенной программе.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>430</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="../../res/res.qrc">
        <normaloff>:/icons/lastfm.png</normaloff>:/icons/lastfm.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Last.FM</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="lastfmUserGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Пользователь</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Пользователь:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lastfmLogin">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Пароль:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lastfmPasswd">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnLastfmCheck">
            <property name="text">
             <string>Проверка</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Скроббинг</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QCheckBox" name="lastfmScrobbingEnable">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Включить скроббинг записей на Last.FM</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Скроббить при прослушивании</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="lastfmScrobbingSize">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>99</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>% от трека</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>242</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
